Sources/PostgresNIO/New/Extensions/ByteBuffer+PSQL.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  File.swift
+//
+//
+//  Created by Fabian Fett on 15.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```